experiments/explore-bigdata-to-smalldata/README.md

```
@@ -0,0 +1,56 @@
+
+В начале скрипта задайте `INSTITUTION_ID` — идентификатор университета в OpenAlex (совпадает с SciSciNet-v2):
+
+```python
+INSTITUTION_ID = "I172901346"  # СПбГУ
+```
+
+ID можно найти на [openalex.org](https://openalex.org/institutions).
+
+## Зависимости
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)